rs/registry/canister/src/mutations/do_update_subnet.rs

```diff
@@ -1461,248 +1340,4 @@ mod tests {
-        expected = "Deprecated fields ecdsa_key_signing_{en,dis}able should not be used \
-        together with chain_key_signing_{en,dis}able."
-    )]
-    fn test_disallow_legacy_and_chain_key_ecdsa_signing_disable_specification_together() {
-        let key_id = EcdsaKeyId {
-            curve: EcdsaCurve::Secp256k1,
-            name: "fake_key_id".to_string(),
-        };
-        let mut registry = invariant_compliant_registry(0);
-
-        let subnet_id = subnet_test_id(1000);
-        let payload = UpdateSubnetPayload {
-            ecdsa_key_signing_disable: Some(vec![key_id.clone()]),
-            chain_key_signing_disable: Some(vec![MasterPublicKeyId::Ecdsa(key_id)]),
-            ecdsa_config: None,
-            chain_key_config: None,
-            ..make_empty_update_payload(subnet_id)
-        };
-
-        registry.do_update_subnet(payload);
-    }
-
-    // TODO[NNS1-3022]: Remove this test.
-    #[test]
-    #[should_panic(
-        expected = "Deprecated fields ecdsa_key_signing_{en,dis}able should not be used \
-        together with chain_key_signing_{en,dis}able."
-    )]
-    fn test_disallow_legacy_enable_and_chain_key_ecdsa_signing_disable_specification_together() {
-        let key_id = EcdsaKeyId {
-            curve: EcdsaCurve::Secp256k1,
-            name: "fake_key_id".to_string(),
-        };
-        let mut registry = invariant_compliant_registry(0);
-
-        let subnet_id = subnet_test_id(1000);
-        let payload = UpdateSubnetPayload {
-            ecdsa_key_signing_enable: Some(vec![key_id.clone()]),
-            chain_key_signing_disable: Some(vec![MasterPublicKeyId::Ecdsa(key_id)]),
-            ecdsa_config: None,
-            chain_key_config: None,
-            ..make_empty_update_payload(subnet_id)
-        };
-
-        registry.do_update_subnet(payload);
-    }
-
-    // TODO[NNS1-3022]: Remove this test.
-    #[test]
-    #[should_panic(
-        expected = "Deprecated fields ecdsa_key_signing_{en,dis}able should not be used \
-        together with chain_key_signing_{en,dis}able."
-    )]
-    fn test_disallow_legacy_disable_and_chain_key_ecdsa_signing_enable_specification_together() {
-        let key_id = EcdsaKeyId {
-            curve: EcdsaCurve::Secp256k1,
-            name: "fake_key_id".to_string(),
-        };
-        let mut registry = invariant_compliant_registry(0);
-
-        let subnet_id = subnet_test_id(1000);
-        let payload = UpdateSubnetPayload {
-            ecdsa_key_signing_disable: Some(vec![key_id.clone()]),
-            chain_key_signing_enable: Some(vec![MasterPublicKeyId::Ecdsa(key_id)]),
-            ecdsa_config: None,
-            chain_key_config: None,
-            ..make_empty_update_payload(subnet_id)
-        };
-
-        registry.do_update_subnet(payload);
-    }
-
-    // TODO[NNS1-3022]: Remove this test.
-    #[test]
-    #[should_panic(
-        expected = "Proposal attempts to enable and disable signing for the same ECDSA keys"
```

Comment:
Just double-checking - we have all the same tests for chain-key already?

Reply:
Yes, this test's chain key cousin is called `enable_and_disable_signing_lists_should_not_have_same_keys_in_single_request`. 

I ported all legacy tests marked `TODO[NNS1-3022]` to the chain key API as part of the original work 8 months ago, that's why you don't see those tests being added in this PR.